locales/help_enu/gkhDateSpecials.html

```diff
@@ -1,87 +1,110 @@
+        place of death. From GEDKeeper&#39;s point of view, all events are the same. GEDKeeper processes them the same way and never limits the number of stored events. For
+        purposes of the next research, a single person record is able to store several death and birth events. This allows storing several versions in case, for example, there
+        is conflicting documentation.
+        This provides significant flexability in the software. You do not have to write such events into the text notes or write birth and death events as range of dates.
+        Moreover, having
+        such an approach allows the user to attach sources of information to any event. When a researcher finds reliable information, the user can delete an incorrect
+        event or mark it as erroneous. The latter allows a history to see that some information, even if it was erroneous, has been rechecked then refuted.
+    </p>
+
+    <p>The number of years in the &quot;Age&quot; and &quot;Life span&quot; columns is calculated as follow:
+
+        <ul>
+            <li>Life span is calculated strictly as the difference between birthday and obituary. If either of these dates is incomplete, nothing is done&#151;the result is undefined.</li>
+            <li>
+                The Age field is calculated as differences of dates between the birth and death events. This is an age when a person has died if a death event has a valid date. Age is not calculated
+                when a death event exists but has no date. Age is calculated using the current date if a date for the death event has not beeen recorded. For example, if
+                a person was born in 1700 and yet has no death event, in 2015 GEDKeeper shows its &quot;Age&quot; as 315 years. To avoid this when a death has occurred,
+                even if you do not know when it happened, always add a death event with an empty date.
+            </li>
+        </ul>
+    </p>
+
+    <p>
+        To make processing of large amounts of data more efficient, GEDKeeper offers a check
+        database service (accessible from the &quot;Service\Tools&quot; menu, &quot;Check database&quot; tab). This service checks all person records for those without a death event.
+        If a calculated age exceeds scientifically proven limits of a human life, the service suggests a
+        fix for a death event where the user may select a suggestion and click the &quot;Fix&quot; button. For GEDKeeper the limit of a human life is 122 years (see the
+        &quot;<a href="https://en.wikipedia.org/wiki/List_of_the_verified_oldest_people" target="_top">List of the verified oldest people</a>&quot; Wikipedia page). Please note
+        that the checking function does not set an obit of a person to the maximum value&#151;it just creates an empty death event as an indicator.
+    </p>
+
+    <hr />
+    <p>See also: <a href="gkhRec_Event.html">Event</a>.</p>
```

Comment:
The file is changed almost completely, no specific change is visible - a solid "flare". It will not work, you need to correct the formatting back.